pyubee/__init__.py

```diff
@@ -314,36 +364,43 @@ def _base_url(self):
         return 'http://{}'.format(self.host)
 
     def _get(self, url, **headers):
-        """Do a HTTP GET."""
-        # pylint: disable=no-self-use
-        _LOGGER.debug('HTTP GET: %s', url)
-        req_headers = {'Host': self.host}
+        if hasattr(self, 'authenticator') and isinstance(self.authenticator, DigestAuthAuthenticator):
+            # We are using digest auth:
+            response = requests.get(url, timeout=HTTP_REQUEST_TIMEOUT, auth=HTTPDigestAuth(self.username, self.password))
+            return response
+        else:
```

Comment:
No need for this else block + indented code, the `return` in the if statement already terminates execution.